lua/lspinstall/servers/ember.lua

```diff
@@ -0,0 +1,9 @@
+local config = require'lspinstall/util'.extract_config('ember')
```

Comment:
You can't do that, there is no 'ember' config in lspconfig.

Reply:
There is one now: https://github.com/neovim/nvim-lspconfig/pull/1042